drivers/cst816s/include/cst816s_internal.h

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (C) 2020 Koen Zandberg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
```

Comment:
There is a missing Doxygen comment here. The following should be added:

```suggestion
/**
 * @ingroup     drivers_cst816s
 *
 * @{
 * @file
 * @brief       Internal constants for CST816S
 *
 * @author      Koen Zandberg <koen@bergzand.net>
 */
```

Reply:
Added, thanks